### PR TITLE
Move "Virtual Spaces" preference to "Editor -> Features"

### LINKED ISSUE #3605: Prefs: move "virtual spaces" to the "Features" tab
As @techee (IIRC) mentioned somewhere, the *Editor→Display* tab is getting crowded to the point it makes the prefs window grow. I suggest moving the *Virtual spaces* section to the *Editor→Features* tab that not only has room, but also seems more fitting to me – it's not a display thing, it's how the editor behaves, isn't it?

### PR DESCRIPTION
**Pull request by b4n**
"Editor -> Display" is very crowded already, and virtual spaces are not merely a display feature, but rather a tool for editing.

Close #3605.

### CODE REVIEW

**Comment by techee** [comment]
Looks good - I just tested this PR and it works as described (no way for a human being to review the XML diff itself).

In the future we could introduce some tab like "Behavior" which is so generic-sounding we'd be free to place there just anything :-).

**Comment by b4n** [comment]
>Looks good - I just tested this PR and it works as described (no way for a human being to review the XML diff itself).

Use Git's own diff to see it, and enable highlighting of moved lines. Almost all of it is just moved, minus a few lines holding indices and such (and maybe uselessly setting an active radio button, I don't remember if I left it in there and am on a phone)

>In the future we could introduce some tab like "Behavior" which is so generic-sounding we'd be free to place there just anything :-).

Hehe, smart 🤓

**Comment by elextr** [comment]
> In the future we could introduce some tab like "Behavior" which is so generic-sounding we'd be free to place there just anything :-).

What, "General" isn't general enough ;-P

File: data/geany.glade [hunk 1]
```diff
@@ -3057,6 +3057,91 @@
...
+                                <property name="can-focus">False</property>
+                                <child>
+                                  <object class="GtkRadioButton" id="radio_virtualspace_disabled">
+                                    <property name="label" translatable="yes">Disabled</property>
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">True</property>
+                                    <property name="receives-default">False</property>
+                                    <property name="tooltip-text" translatable="yes">Do not show virtual spaces</property>
+                                    <property name="use-underline">True</property>
+                                    <property name="active">True</property>
```
Comment (eht16) [inline data/geany.glade:3083]: This seems new but probably added by Glade automatically and we set an explicit default value in the code anyway, so it's fine.

**Review by eht16 — APPROVED** [review]
LGTM

**Pull request merged**